packages/flutter/lib/src/widgets/image.dart

```diff
@@ -44,6 +45,15 @@ ImageConfiguration createLocalImageConfiguration(BuildContext context, { Size si
   );
 }
 
+/// Prefetches an image into the image cache.
+Future<Null> prefetchImage(ImageProvider provider, BuildContext context, { Size size }) {
```

Comment:
"prefetch" implies that something is being read from network or disk. How about "waitForImage".

Why a top-level function and not a method on `ImageProvider`?